Assets/Scripts/Augment/AugmentImplementations/AmmoBox.cs

```diff
@@ -24,6 +26,23 @@ private void Start()
                 .setLoopType(LeanTweenType.easeInOutCubic).setLoopCount(-1));
     }
 
+    private void OnEnable()
+    {
+        ammoBoxes.Add(this);
+    }
+
+    private void OnDisable()
+    {
+        ammoBoxes.Remove(this);
+    }
+
+    public static AmmoBox GetClosestAmmoBox(Transform from)
```

Comment:
This method could take a position vector, it doesn't use anything but `from.position`

Reply:
Done 👌